gvm/protocols/gmpv7/gmpv7.py

```diff
@@ -4783,8 +4782,6 @@ def modify_config_set_family_selection(
             families: List of NVT family names to select.
```

Comment:
The comment needs to be updated too

Reply:
True. Do you think we need to set a default True for this too? like auto_add_new_families?

Reply:
Currently I don't know how that should be possible easily. In that case you would need to check each item in the list if it is a tuple and if the second value in the tuple is available.

Reply:
wouldn't be that much work, I guess?

Reply:
You should at least test for tuples like `('foo', )` and `('foo', None)` and `('foo', 'bar')` ... and throw some InvalidArgument errors?